app/views/layouts/application.html.erb

```diff
@@ -21,18 +21,17 @@
 <% if user_signed_in? %>
   <%= content_for :navbar_right do %>
     <div class='session-details-panel'>
-      <span class='session-details-panel__username'>Logged in as: <%= current_user.name %></span>
-      <%= link_to 'Logout', destroy_user_session_path %>
-      <%= link_to 'Change password', edit_user_registration_path %>
+      <%= link_to current_user.name, Plek.current.find('signon') %>
```

Comment:
This requires the GovukAdminTemplate to be updated and breaks the error pages. As a result I would prefer to do this as a separate PR once this one is mreged. :)